### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (9 - "users")

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#488]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

6)	The use of the word “users” within a provision
This is a problem we have faced in all of the standards groups that I have worked on dealing with accessibility.   We found that we continually had a tendency to inject the word "users" or “users can” into our normative provisions.  This only caused us to later have to go back and rework them to get them back out.  
For example, any normative statement that says that something “should be done in order for users to” runs into the problem of not defining which users one is talking about.   For example, “Alt text should be provided that will allow users to understand the meaning of the image”.  Which users?  We cannot be talking about every user in the world, because there will always be users who are going to be completely unable to use whatever technology were talking about.  
For example, diagrams on a website on quantum physics or string theory are not going to be used or understood by a large number (most) users no matter what alt text you provide – because they couldn’t understand it if they could see the diagrams. The same thing about advanced cooking, or woodworking or a lot of other things.  There will always be websites that are talking about things that are more advanced and require more advanced knowledge than a user has.  There's nothing one can do to that website that is going to make it understandable to that individual.  Even advanced knitting sites.
But the same goes for shopping sites.  There are some (a much smaller number) who will be confused by any shopping site – and would be confused if you left them unattended in any physical store as well. 
We can’t mean to require that all users will be able to understand.  You also can't define “users” as people who are already using it, or else that's a self-defeating requirement.   You can define it as “all those who ‘could’ “  since you have the same circular problem.
Having said this about not including “users” in the actual provisions, I will note that including them in the RATIONALE for a provision is fine and very useful. 
For example 
A text alternative shall be available for all non-text content. 
Rationale: This allows the non-text content to be understood since the text alternative can be rendered into visual, auditory, or tactile form – or translated into different languages or symbol sets to meet the needs of individual user – just like the text portion of the website can.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.
